### ISSUE TITLE
OnDemandInstance shown as ReservedInstanceHeavy when Custom Tags enabled

### ISSUE THREAD

**Issue by anpbabaki** [Netflix/ice#189]
All OnDemandInstances are being shown as ReservedInstanceHeavy on the pages using Resource Groups.  Running the latest commit, Grails 2.4.4

Here you can see us-east-1 EC2 costs are 15,502.69 and the On Demand and Reserved are properly broken down

![image](https://cloud.githubusercontent.com/assets/14282717/9861896/44a268ac-5b03-11e5-9be7-a38f36cbfd8c.png)

Here you see on the Details with Rsource Groups, they are lumped into one Operation.

![image](https://cloud.githubusercontent.com/assets/14282717/9861920/8269174e-5b03-11e5-9812-36db2f7dfcd3.png)

**Comment by anpbabaki** [comment]
anyone else seeing this?

**Comment by harishbsrinivas** [comment]
Yes seeing the same on a fresh install. Used the ansible scripts from http://answersforaws.com/code/netflixoss/ice/ to set it up.

**Comment by blee1170** [comment]
We are having this exact problem as well. Has anyone figured it out?

**Comment by blee1170** [comment]
After a few days of digging I think I have it tracked to a undocumented config setting. In the code it defaults the property ice.resourceGroupCost to "modeled", which the code then goes and creates the ReservedInstanceHeavy even if you are not using it.

I set the property to ice.resourceGroupCost=fishy and the ReservedInstance was removed from the list.

I don't know if this will exactly fix your problem, but I have a feeling it might.

**Comment by anpbabaki** [comment]
This post pretty much explains the reasoning for the behavior we are seeing

https://groups.google.com/forum/#!topic/iceusers/zh6pBh0c7oA
